tracer/test/Datadog.Trace.Tests/LambdaRequestBuilderTests.cs

```diff
@@ -0,0 +1,80 @@
+// <copyright file="LambdaRequestBuilderTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+
+using Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS;
+using Datadog.Trace.TestHelpers;
+
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace Datadog.Trace.Tests
+{
+    public class LambdaRequestBuilderTests
+    {
+        [Fact]
+        [Trait("Category", "ArmUnsupported")]
```

Comment:
I don't think these need the `[Trait("Category", "ArmUnsupported")]` trait? They're just using c# at this point right?

Reply:
Oh right, I forgot about this. These don't rely on containers or other architecture-dependent assemblies so we should be able to remove this trait

Reply:
✅